crates/ruma-common/src/time.rs

```diff
@@ -28,6 +28,16 @@ impl MilliSecondsSinceUnixEpoch {
     pub fn to_system_time(self) -> Option<SystemTime> {
         UNIX_EPOCH.checked_add(Duration::from_millis(self.0.into()))
     }
+
+    /// Get the time since the unix epoch in milliseconds.
+    pub fn get(&self) -> UInt {
+        self.0
+    }
+
+    /// Get time since the unix epoch in seconds.
+    pub fn as_secs(&self) -> UInt {
+        self.0 / UInt::from(1000 as u32)
```

Comment:
This is what the `uint!` macro that I suggested before is for – creating `UInt` from literal with fewer characters. Just import it from `js_int`.

Reply:
i didn't see that you suggested that. :)